Dockerfile

```diff
@@ -16,6 +16,8 @@ RUN apt-get install -y -qq wget
 # install jq to parse json within bash scripts
 RUN apt-get install -y jq
 
+RUN apt-get -y -qq update && apt-get -y -qq upgrade
+
```

Comment:
@MukuFlash03 is there a reason why this is not actually just a upgrade of the base image either?

```
FROM ubuntu:jammy
```

again, **you want to avoid blanket updates** in the dockerfile

Reply:
I am merging this for now to fix the security vulnerabilities, but I would like to see some discussion around this.